### ISSUE TITLE
AutoLayout Issue

### ISSUE THREAD

**Issue by croossin** [Yalantis/Segmentio#2]
Keep crashing on setup...

```
Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '*** +[NSLayoutConstraint constraintWithItem:attribute:relatedBy:toItem:attribute:multiplier:constant:]: A multiplier of 0 or a nil second item together with a location for the first attribute creates an illegal constraint of a location equal to a constant. Location attributes must be specified in pairs'
```

After checking with debugger it looks like it crashes on line 196 in `Segmentio.swift` when function `setupHorizontalSeparator()` is called.... Any thoughts?

**Comment by serejahh** [comment]
Hi @croossin! Thanks for reporting the issue. Could you describe steps to reproduce the bug?

**Comment by croossin** [comment]
Just followed the simple steps in the README... (thanks for making them simple btw)!

    let segmentioViewRect = CGRect(x: 0, y: 0, width: UIScreen.mainScreen().bounds.width, height: 125)

    let segmentioView = Segmentio(frame: segmentioViewRect)

    segmentioView.setup(content: content(), style: SegmentioStyle.ImageOverLabel, options: nil)

    view.addSubview(segmentioView)

**Comment by croossin** [comment]
If I comment out that line `setupHorizontalSeparator()` it will run and display but in a very odd format.. See attached screenshot

![simulator screen shot aug 28 2016 10 51 00 am](https://cloud.githubusercontent.com/assets/7241069/18035589/64b6eb82-6d0d-11e6-8865-b1fdbb8b14c8.png)

**Comment by serejahh** [comment]
It would be great if you could provide the demo project with the crash

**Comment by croossin** [comment]
Pushed a temp proj [here](https://github.com/croossin/Resty)

**Comment by serejahh** [comment]
Thanks a lot! I'm sorry, I'll investigate the issue tomorrow, I'm still on weekend :)

**Comment by croossin** [comment]
No worries! Thanks for the speedy response 👍

**Comment by serejahh** [comment]
According to the error message sigmentio's superview is nil. It happens because `setup(content: style:options:)` setups constraints based on its superview. In your project you call setup method while sigmentio doesn't have a superview yet. I suppose a fix can look like:

``` swift
public override func didMoveToSuperview() {
    super.didMoveToSuperview()
    
    if superview != nil {
        setupHorizontalSeparator()
    }
}
```
instead of calling `setupHorizontalSeparator()` in that setup method (line 196). 

I'll create a pull request to the author. It the solution is good, I'll let you know when it's merged

**Comment by serejahh** [comment]
The fix is merged to `1.0`. Please, update

**Comment by croossin** [comment]
It is all fixed 👍 

Thank you for the speedy fix!
